bayesian/utils.py

```diff
@@ -1052,13 +1052,19 @@ def get_recommendation_feedback_by_ecosystem(ecosystem):
             filter(RecommendationFeedback.stack_id == StackAnalysisRequest.id).all()
         result = []
         for feedback in feedback_list:
+            if feedback.stack_request.dep_snapshot.result is None:
+                dependencies = []
+            else:
+                dependencies = feedback.stack_request.dep_snapshot.result.details._resolved
```

Comment:
If `dep_snapshot` is a dictionary, then this won't work.

`feedback.stack_request.dep_snapshot.get('result', [{}])[0].get('details', [{}])[0].get('_resolved', [])`

^^ that should do the trick, IMO. But I haven't tested it.